release.config.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
           'e2e': {
             'npmPublish': false,
             'pkgRoot': 'packages/e2e'
+          },
+          'cli': {
+            'npmPublish': false,
+            'pkgRoot': 'packages/e2e'
```

Comment:
wrong package root

Reply:
And we need to add the cli package.json path in line 90 `'@semantic-release/git',`

Reply:
cool, what does `@semantic-release/git`  do ?

Reply:
Makes a commit into `main` with the changelog and all package.json updated. e.g https://github.com/lightdash/lightdash/commit/680efdd95ec6abcd64e36b8f8a8887de2e31cc83